src/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
-if (MSVC)
-    set(CMAKE_CXX_FLAGS_DEBUG "/ZI /Od" CACHE STRING "" FORCE)
-else ()
-    set(CMAKE_CXX_FLAGS "-fpermissive")
+if (NOT MSVC)
+    add_compile_options(-fpermissive)
```

Comment:
Lets drop `-fpermissive` too. I doubt there is any need in such tricks. If it still exists, better to fix that code.